helpers/index.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  expectThrow: async promise => {
+    try {
+      await promise
+    } catch (error) {
+      // console.warn(error.message)
+      // TODO: Check jump destination to destinguish between a throw
```

Comment:
Please remove this TODO. Recent version of solidity use an `invalid opcode` for throw. https://solidity.readthedocs.io/en/develop/control-structures.html?highlight=exceptions#error-handling-assert-require-revert-and-exceptions